src/versionbits.cpp

```diff
@@ -27,6 +27,12 @@ ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex*
     int64_t nTimeStart = BeginTime(params);
     int64_t nTimeTimeout = EndTime(params);
 
+    // Check if this deployment has a fixed activation height.
+    if (nTimeStart < 0) {
+        if (pindexPrev && pindexPrev->nHeight + 1 >= -nTimeStart) return THRESHOLD_ACTIVE;
```

Comment:
paranoid nit: check for ```nTimeStart != std::numeric_limits<int64_t>::min()```

Reply:
This logic has been simplified, so I don't think this is necessary anymore.